sg.tf

```diff
@@ -28,26 +23,17 @@ resource "aws_security_group" "sg-lb" {
   }
 
   tags = {
-    "Name" = "sg-lb"
+    "Name" = "Load Balancer Security group"
   }
 }
 
 //security group for ec2 instances
-resource "aws_security_group" "sg-ec2" {
-  vpc_id = aws_vpc.test-vpc.id
+//TODO delete everything except 22
+//add inbound rule from application load balancer
+resource "aws_security_group" "sg_ec2" {
```

Comment:
Rename to `aws_security_group.nginx_ec2`

Reply:
renamed ec2 security group from sg_ec2 to nginx_ec2